store/sqlstore/user_store.go

```diff
@@ -1055,20 +1045,41 @@ func (us SqlUserStore) PermanentDelete(userId string) store.StoreChannel {
 	})
 }
 
-func (us SqlUserStore) AnalyticsUniqueUserCount(teamId string) store.StoreChannel {
+func (us SqlUserStore) Count(options model.UserCountOptions) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
-		query := ""
-		if len(teamId) > 0 {
-			query = "SELECT COUNT(DISTINCT Users.Email) From Users, TeamMembers WHERE TeamMembers.TeamId = :TeamId AND Users.Id = TeamMembers.UserId AND TeamMembers.DeleteAt = 0 AND Users.DeleteAt = 0"
+		query := sq.Select("COUNT(Users.Id)").From("Users")
+
+		if !options.IncludeDeleted {
+			query = query.Where("Users.DeleteAt = 0")
+		}
+
+		if options.IncludeBotAccounts {
+			if options.ExcludeRegularUsers {
+				query = query.LeftJoin("Bots ON Users.Id = Bots.UserId").Where("Bots.UserId IS NOT NULL")
```

Comment:
This amounts to just a regular join, so you can `s/LeftJoin/Join` here and discard the `Where` condition altogether.